### DISCUSSION TITLE
have to pedal a lot and quickly until the PAS assistance kicks in. HELP

### DISCUSSION THREAD

**Discussion by Benix6** [danielnilsson9/bbs-fw#192] [Q&A] [answered]
Hello - first of all, thank you for the great work. I was looking for software that would allow me to operate my Bafang BBSHD with a 52v 20a battery. The standard configuration solution only supports up to 48v.

My uncle has a 48v 20a battery installed, and his bike works better with the standard configuration software.

Now, onto my problem. I feel like I have to pedal a lot and quickly when starting off until the PAS assistance kicks in, and then the bike shoots forward. This is particularly problematic when going uphill, and you have to come to a stop for various reasons. Then you can basically only start with the throttle lever. With my uncle's bike, the drive engages earlier, and the motor assistance/acceleration is "smoother".

I haven't found a setting like "Assist start at %", there's only an option for Start delay in degrees. I have no idea what that means; even if I set it to 0 degrees, I still need a few rotations before the motor assists. Is there a specific setting for this, or could danielnilsson9 possibly create an option for it?

Best regards,
Benji

**Comment by kb79000** [comment]
> Hello - first of all, thank you for the great work. I was looking for software that would allow me to operate my Bafang BBSHD with a 52v 20a battery. The standard configuration solution only supports up to 48v.
> 
> My uncle has a 48v 20a battery installed, and his bike works better with the standard configuration software.
> 
> Now, onto my problem. I feel like I have to pedal a lot and quickly when starting off until the PAS assistance kicks in, and then the bike shoots forward. This is particularly problematic when going uphill, and you have to come to a stop for various reasons. Then you can basically only start with the throttle lever. With my uncle's bike, the drive engages earlier, and the motor assistance/acceleration is "smoother".
> 
> I haven't found a setting like "Assist start at %", there's only an option for Start delay in degrees. I have no idea what that means; even if I set it to 0 degrees, I still need a few rotations before the motor assists. Is there a specific setting for this, or could danielnilsson9 possibly create an option for it?
> 
> Best regards, Benji

Hi test 75 degrees

**Reply by Benix6** [reply]
can you explain how start delay ° works?
and I already tried the standard configuration

**Reply by kb79000** [reply]
https://github.com/danielnilsson9/bbs-fw/wiki/Configuration-Tool

**Reply by Benix6** [reply]
yes, but what does "in degress" mean?
is this the distance the pedal needs to make?

**Reply by dav0000000** [reply, marked as answer]
There are two requirements that affect the way pas turns the motor on. 

The degrees is the amount of rotation required. 90 degrees is a quarter rotation of the pedals. There are 360 degrees for a full rotation.

The other is stop delay. This is the maximum time between pas pulses detected by the pas sensor. The pas sensor has 24 magnets. In one full rotation of the pedals the sensor will see 24 pulses. When this is set to 200ms it means you have to pedal faster than 12.5rpm at the crank for the motor to start. 200ms x 24 pulses = 4800ms. 12.5rpm is 4800ms per pedal rotation. If there are 60000ms per minute then you see that 60000 / 4800 = 12.5rpm. That should be sufficient. I use 350ms stop delay in case I get stuck on a hill.

**Reply by EigenMania** [reply]
Ahhhh @dav0000000 thanks a bunch for explaining the stop delay clearly (though I didn't ask). I knew somewhere it was probably possible to change the minimum PAS cadence but I never got around to figuring it out XD

**Reply by Benix6** [reply]
Thanks @dav0000000 for that great explaination! That‘s something I can work with, thanks a lot

**Reply by Benix6** [reply]
@dav0000000 that's it.
I have set the Start Delay to 15° and the Stop Delay to 150. Feels nice - thanks a lot.

At the beginning, I set the stop delay to the minimum, 50, because I like to have immediate feedback. This means that if you stop pedaling during the ride, the motor continues to run for a short while. The lower the value, the less it runs. Now, I've made a small trade-off by setting it to 150, which works well for me. @danielnilsson9, is it possible to expand this through software? For example, having a higher value for the start compared to the motor coasting afterwards? Thanks in any case, I'm now super satisfied.
